<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 60, 68)</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>19</y>
       <width>661</width>
       <height>641</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;background-color:rgb(31, 31, 31)</string>
     </property>
     <widget class="QLabel" name="label_87">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>351</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/Screenshots/Screenshot 2024-07-02 181839.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_88">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/Screenshots/Screenshot 2024-06-30 223013.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_83">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Thông tin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>581</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(57, 60, 68)</string>
      </property>
      <property name="text">
       <string>Gà chiên nước mắm là một món ăn truyền thống và rất được ưa chuộng trong ẩm thực Việt Nam. Món ăn này nổi bật nhờ sự kết hợp tuyệt vời giữa vị giòn rụm của gà chiên và hương vị đậm đà của nước mắm, tạo nên một hương vị đặc trưng, hấp dẫn. Dưới đây là một số thông tin chi tiết về món gà chiên nước mắm:

Lịch sử và nguồn gốc:
Gà chiên nước mắm có nguồn gốc từ ẩm thực dân gian Việt Nam. Nước mắm, một trong những nguyên liệu chính, là một loại gia vị truyền thống đã xuất hiện từ rất lâu trong nền văn hóa ẩm thực Việt.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_return_gachien_book">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>570</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>D:/download/tải_xuống-removebg-preview.png</normaloff>D:/download/tải_xuống-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>D:/download/chef.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
